cmake/configure_orion.intel.cmake

```diff
@@ -1,2 +1 @@
 set(INLINE_POST     ON  CACHE BOOL "Enable inline post" FORCE)
```

Comment:
inline post is not available for regional fv3, maybe you don't need it compiled in the model as well

Reply:
> inline post is not available for regional fv3, maybe you don't need it compiled in the model as well

Good point.

Reply:
Done, removed it for both gnumake and cmake on all machines.